pyglet/event.py

```diff
@@ -401,15 +409,29 @@ def dispatch_event(self, event_type, *args):
         return False
 
     @staticmethod
-    def _raise_dispatch_exception(event_type, args, handler, exception):
-        # A common problem in applications is having the wrong number of
```

Comment:
This should be left as a block comment. It's an internal method that is not intended to be seen by users. The block comment contains notes for future programmers, not intended for user documentation.